external/fv3fit/fv3fit/keras/_models/shared/pure_keras.py

```diff
@@ -44,12 +46,14 @@ def __init__(
             model: keras model to wrap
             unstacked_dims: if given, stacking should leave these dimensions in place.
                 by default uses z-dimension names from fv3gfs.util
+            n_halo: if given, number of halo points required in input data
```

Comment:
Removed "if given", since this is always given and the default is written in the keyword argument. Should only need to state the true default in the docstring when the kwarg says None.